src/validation/decorators/is_kolibri_trigger_t.ts

```diff
@@ -28,14 +28,18 @@ import {
 import { KolibriError, errorcode } from '../../common/errorcode';
 
 
+export function isKolibriNodeTriggerT(value: any) {
+    if (!isInt(value) || !min(value, 0) || !max(value, 4294967295)) {
```

Comment:
Personally I like it more if we instead of having a magic number pass a constant, but it's up to you in case of 4294967295

Reply:
Good point